### PR TITLE
Allows to use IAvaloniaObjects where a IControl is currently required

### PR DESCRIPTION
**Pull request by JaggerJo**

### CODE REVIEW

File: src/Avalonia.FuncUI.DSL/AutoCompleteBox.fs [hunk 1]
```diff
@@ -15,62 +15,62 @@ module AutoCompleteBox =
     
     type AutoCompleteBox with
         static member onSelectionChanged<'t when 't :> AutoCompleteBox>(func: SelectionChangedEventArgs -> unit, ?subPatchOptions) =
-            AttrBuilder<'t>.CreateSubscription<_>(AutoCompleteBox.SelectionChangedEvent, func, ?subPatchOptions = subPatchOptions)
+             AttrBuilder.CreateSubscription<'t, _>(AutoCompleteBox.SelectionChangedEvent, func, ?subPatchOptions = subPatchOptions)
```
Comment (AngelMunoz) [inline src/Avalonia.FuncUI.DSL/AutoCompleteBox.fs:18]: unwanted extra space?

Comment (JaggerJo) [reply src/Avalonia.FuncUI.DSL/AutoCompleteBox.fs:18]: yep, good catch

**Comment by JaggerJo** [comment]
#86

**Comment by sleepyfran** [comment]
I believe this is what I did as part of #220? @JaggerJo are we missing any other part or can we close this PR? 😃

**Pull request closed without merging**